nifi-nar-bundles/nifi-hbase-bundle/nifi-hbase-processors/src/test/java/org/apache/nifi/hbase/MockHBaseClientService.java

```diff
@@ -131,4 +131,48 @@ public void setThrowException(boolean throwException) {
     public byte[] toBytes(final String s) {
         return s.getBytes(StandardCharsets.UTF_8);
     }
+
+    @Override
+    //Implementation of this and the isHexDigit and toBinaryFromHex are from HBase Bytes.java
+    public byte[] toBytesBinary(String s) {
```

Comment:
I'm wondering if we should declare a test dependency on HBase 1.1.2 so that we don't need to copy this code, or if its better/easier to just copy this code we might need to put something in our notice or license to attribute it